tmexp/create_bow.py

```diff
@@ -44,15 +44,20 @@ def create_bow(
 ) -> None:
     logger = create_logger(log_level, __name__)
 
-    check_exists(input_path)
-    output_dir = os.path.join(output_dir, dataset_name)
+    if dataset_name is None:
```

Comment:
Sorry I was not clear on this point: we have 2 different cases for the `dataset_name` arg: one can have a default (`preprocess`), the other cannot and should be required (`create_bow`, since the name should already exist from existing output). It means we should add an argument to `add_dataset_arg` to know if we have `required=True` or if we have the current `default=…`. If we have those two different parser definitions, we don't need to check for `None` here and we're also certain that the user does not use a default by mistake.

Reply:
ah yes my bad, I forgot that part, updating and merging